device-types/Datacom/DM4270-8XS-plus-16VS-plus-6CX.yaml

```diff
@@ -0,0 +1,117 @@
+---
+manufacturer: Datacom
+model: DM4270 8XS+16VS+6CX
+slug: datacom-dm4270-8xs-plus-16vs-plus-6cx
+part_number: 800.5313.xx
+u_height: 1.0
+is_full_depth: true
+airflow: side-to-rear
+comments: Switch L2 L3 MPLS, wire speed, with 8 optical 10GE ports on SFP+ connectors, 16 optical 25GE 10GE ports on SFP28 connectors, and 6 multi-rate
+  optical 100GE 40GE ports on QSFP28 connectors.  1U metal chassis for 19 rack with 2 fan modules. Power supplies, MPLS license, and optical modules must
+  be purchased separately.
+weight: 6.28
+weight_unit: kg
+console-ports:
+  - name: Console0
+    type: rj-45
+  - name: Console1
+    type: usb-mini-b
+console-server-ports:
+  - name: Storage
+    type: usb-a
```

Comment:
Is this actually used as a console server port? Or is it just for the switch to access a USB storage device? If the latter, please remove this definition as it would be out of scope.

Reply:
removed